t/04-scheduler.t

```diff
@@ -24,13 +24,15 @@ use OpenQA::Resource::Jobs;
 use OpenQA::Constants qw(WEBSOCKET_API_VERSION DB_TIMESTAMP_ACCURACY);
 use OpenQA::Jobs::Constants;
 use OpenQA::Test::Database;
+use OpenQA::Test::Utils 'setup_mojo_app_with_default_worker_timeout';
 use OpenQA::Utils 'assetdir';
 use Test::Mojo;
 use Test::MockModule;
 use Test::Warnings ':report_warnings';
 use OpenQA::Schema::Result::Jobs;
 use OpenQA::WebAPI::Controller::API::V1::Worker;
 
+setup_mojo_app_with_default_worker_timeout;
```

Comment:
I wonder if this affects the reliability of this test - I was just investigating a case where `t_seen` was [off by a second in CircleCI](https://app.circleci.com/pipelines/github/os-autoinst/openQA/4196/workflows/bbc95dc4-cb0f-4426-82b0-826d3fce4848/jobs/40093).

Reply:
In the failures you're referring `t_seen` has been updated unexpectedly. Not sure why that's the case but I doubt this line changes that.